assets/src/blocks/Articles/ArticlesBlock.js

```diff
@@ -131,7 +131,6 @@ export class ArticlesBlock {
         },
         exclude_post_id: {
           type: 'integer',
-          default: ''
         },
       },
       // withSelect is a "Higher Order Component", it works as
```

Comment:
This comment is also from and old sample, please remove it while you update this.